CYBRHardeningCheck/Main.ps1

```diff
@@ -542,6 +564,17 @@ $outputFile = New-HTMLReportOutput -machineName $machineName -components $(Get-D
 Out-HardeningFolderPath -Path $outputFile -TotalComponentsFound $(Get-DetectedComponents).Name.Count
 
 Write-LogMessage -Type Info -MSG "Hardening Health Check Report located in: $outputFile" -LogFile $LOG_FILE_PATH
-. $outputFile
+# . $outputFile
+
+    if($ReturnReportPath.IsPresent)
+    {
+        # return the report path and skip opening the file interactively
+        EndScript -ReturnData -HardeningReportPath $outputFile
```

Comment:
@ediulia it seems a bit redundant to use the same switch twice
I don't see why adding it to the EndScript function